content/kb/tutorials/databases/snowflake.md

```diff
@@ -78,15 +77,13 @@ Your local Streamlit app will read secrets from a file `.streamlit/secrets.toml`
 ```toml
 # .streamlit/secrets.toml
 
-[connections.snowpark]
+[connections.snowflake]
 account = "xxx"
 user = "xxx"
 password = "xxx"
-role = "xxx"
 warehouse = "xxx"
 database = "xxx"
 schema = "xxx"
-client_session_keep_alive = true
 ```
```

Comment:
Why remove these? I don't think they should be removed. I think role is required and we want to encourage / remind people to use client_session_keep_alive

Reply:
They weren't listed in the [snowflake-connector-python documentation](https://docs.snowflake.com/en/developer-guide/python-connector/python-connector-example#connecting-using-the-default-authenticator).  I tried to match what was explained there and didn't see any mention in the tutorial of those two parameters otherwise. But I don't know the internals, so can definitely put them back.

Reply:
Ya let's leave them in.